circle.yml

```diff
@@ -481,6 +481,11 @@ jobs:
           working_directory: ~/build
           command: |
             bin/evmone-statetest ~/tests/EIPTests/StateTests/stEOF
+      - run:
```

Comment:
Please change test revision for now to `update-tests` (on line 477), that's the latest version